src/reducers/collection.js

```diff
@@ -48,7 +49,7 @@ function createDefaultStatus(schema, tag, settings = {}) {
       schema,
       tag,
       type: 'collection',
-      id: _.uniqueId(),
+      id: uuidv4(),
```

Comment:
This will reproduce something like this: '8fb49db5-033b-4229-ab99-1c9edd76f488', we can make our own method to generate UUID and remove '-' from the generated UUID. Probably not necessary as this id is used only for Rio internal stuff...